index.html

```diff
@@ -570,6 +570,32 @@ <h2 id="constrainable-properties">
                 again.</p>
               </td>
             </tr>
+            <tr id="def-constraint-suppressLocalAudioPlayback">
+              <td><dfn>suppressLocalAudioPlayback</dfn></td>
+              <td>{{ConstrainBoolean}}</td>
+              <td>
+                <p>As a setting, this value indicates whether or not the user
+                agent is applying <a>local audio playback suppression</a> to
+                the source.</p>
+                <p>As a constraint, this value is only meaningful if the user
+                selects capturing a [=browser=] [=display surface=]. In that
+                case, a value of <code>true</code> indicates that the user
+                agent SHOULD perform <a>local audio playback suppression</a>
+                on the captured [=browser=] [=display surface=].</p>
+                <p>When <dfn>local audio playback suppression</dfn> is applied,
+                the user agent SHOULD stop relaying audio to the local speakers,
+                but that audio MUST still captured by any ongoing audio-capturing
+                capture-sessions.</p>
+                <p>When a [=browser=] [=display surface=] is subject to multiple
+                concurrent captures, <a>local audio playback suppression</a>
+                SHOULD be active as long as at least one active audio-capturing
+                capture-session remains which specified
+                <a>suppressLocalAudioPlayback</a>.
+                (Note that capture-sessions for which the user did not permit
+                audio-capture are not considered audio-capturing for this
+                purpose.)</p>
```

Comment:
I think the word "active" in "active audio-capturing" above covers this, so we don't need this, I don't think.